client/components/domains/contact-details-form-fields/index.jsx

```diff
@@ -513,9 +513,9 @@ export class ContactDetailsFormFields extends Component {
 		return (
 			<div className="contact-details-form-fields__row">
 				<Input
-					label={ this.props.translate( 'Alternate Email Address' ) }
-					{ ...this.getFieldProps( 'alternate-email', {
-						customErrorMessage: this.props.contactDetailsErrors?.alternateEmail,
+					label={ this.props.translate( 'Alternate email address' ) }
+					{ ...this.getFieldProps( 'email', {
+						customErrorMessage: this.props.contactDetailsErrors?.email,
 					} ) }
```

Comment:
We could probably remove all G Suite/Google Workspace related form fields from this file altogether, since it only seems to be used on `/domains/manage/:domain/edit-contact-info/:site_slug` (and not on the checkout page).

However, in the interest of being efficient with our time, I think it might be better to simply leave this as is for now (even no instance of this component seems to use the Google-related form fields).